compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/window/WindowExceptionHandler.kt

```diff
@@ -0,0 +1,15 @@
+package androidx.compose.ui.window
+
+import androidx.compose.ui.ExperimentalComposeUiApi
+
+/**
+ * Catch exceptions during rendering frames, handling events, or processing background Compose operations.
+ */
+@ExperimentalComposeUiApi
```

Comment:
Isn't this change too much of a new feature? I personally like it, but feel that it may introduce confusion as people will try to use it in common code and it will break on Android. Maybe make it desktop-only for now?

Reply:
Or, right, thanks :).

I wanted to move it to skikoMain